docs/index.md

```diff
@@ -18,13 +18,14 @@
     <img src="https://img.shields.io/github/watchers/ray-project/kuberay?style=social" alt="github watchers"/>
     <img src="https://img.shields.io/github/stars/ray-project/kuberay?style=social" alt="github stars"/>
     <img src="https://img.shields.io/github/forks/ray-project/kuberay?style=social" alt="github forks"/>
-    <a href="https://hub.docker.com/r/kuberay/operator/">
```

Comment:
We’ve been using Quay for a long time, so the numbers on Docker Hub are meaningless.